### ISSUE TITLE
unexpected behaviour of --interval flag

### ISSUE THREAD

**Issue by ravitej-aluru** [alexei-led/pumba#162]
Pumba version 0.7.2 - 92ecb8b () 2020-03-17_09:43_GMT

**Command:** `docker run -d --rm  -v /var/run/docker.sock:/var/run/docker.sock gaiaadm/pumba --interval=23m --log-level=debug kill --signal=SIGKILL peerassembly_peer-node_1 >> ~/log/pumba/peerassembly_peer-node_1_20200905_1920.log 2>&1`

**Expected result:** The container named will be killed in 23 minutes and every 23 minutes there on.

**Actual result:** The container is killed immediately and every 23 minutes there on.

**Comment by alexei-led** [comment]
@ravitej-aluru this is by design - running chaos command immediately and repeat once in a while

**Comment by ravitej-aluru** [comment]
@alexei-led, thanks for your response. When I started using pumba a few months ago, I thought I saw the behaviour I was expecting. Have you changed it recently or was it always like that? I was staggering the killing/stopping of containers of the same service by starting two pumba containers with different intervals on each.

My use case is, I don't want to kill/stop/pause the target containers immediately, but after a certain time period. Is there an option in pumba to add this delay so that pumba doesn't execute the chaos command immediately?

Of course, I can set the command in the docker-compose file to "sleep xx; pumba ..........", but it is a bit weird and not clean. I think it will be a good feature to add to pumba. An option to set initial delay if wanted.
